Base/home/anon/.config/PixelPaint.ini

```diff
@@ -1,12 +0,0 @@
-[PixelGrid]
-Threshold=15
```

Comment:
It seems like some options, like the pixel grid threshold aren't even configurable in the GUI. The only way to know about this option now is to read the source code, which I think isn't ideal.

Making programs write default config options to the .ini files would definitely help in these cases.

Reply:
But as I said, I think it's fine as-is for now.